api/upload.py

```diff
@@ -23,6 +23,7 @@
     'packfile'   : pl.PackfilePlacer,   # Upload N files as a new packfile to a container.
     'labelupload': pl.LabelPlacer,
     'uidupload'  : pl.UIDPlacer,
+    'analysis' : pl.AnalysisPlacer    # Upload N files for an analysys (no db updates)
```

Comment:
There's a documentation table a few lines below this in `process_upload`, that has a feature matrix for each placer.
Could you add a column to that table for the analysis placer?